swift-pipeline

```diff
@@ -157,54 +143,6 @@ if __name__ == "__main__":
     if args.debug:
         from tqdm import tqdm
 
-        # Create a function to print out script runtime statistics as a table
-        def print_script_runtime_statistics(script_runtimes):
-
-            # Init variable to compute the sum of scripts' runtimes
-            total_runtime = 0.0  # sec
-
-            # Add runtimes of all scripts to the sum
-            for script_name, script_runtime in script_runtimes.items():
-                if script_name != "Wallclock time":
-                    total_runtime += script_runtime
-
-            # Create a key-value pair for the total runtime of the scripts
-            script_runtimes["Total CPU time"] = total_runtime
-
-            # Compute the width of first column in the table (add 3 for padding)
-            col_width = max(len(name) for name in script_runtimes.keys()) + 3
-
-            # Set the width of the columns
-            first_item_width, second_item_width, third_item_width = col_width, 20, 27
-
-            # Make zeroth row
-            first_item = "Script name".ljust(first_item_width)
-            second_item = "Run time (s)".ljust(second_item_width)
-            third_item = "Fraction of total CPU time".ljust(third_item_width)
-
-            # Print out table header including the zeroth row
-            print()
-            print("Script Run Time Statistics: \n")
-            print(f"{first_item}{second_item}{third_item}")
-            print("-" * (first_item_width + second_item_width + third_item_width))
-
-            # Create and print out the remaining rows
-            for script_name, script_runtime in script_runtimes.items():
-
-                # Fraction of the total runtime per script
-                run_time_fraction = script_runtime / script_runtimes["Total CPU time"]
-
-                first_item = f"{script_name}".ljust(first_item_width)
-                second_item = f"{script_runtime:.4f}".ljust(second_item_width)
-                third_item = f"{run_time_fraction:.6f}".ljust(third_item_width)
-
-                # Print row
-                print(f"{first_item}{second_item}{third_item}")
-
```

Comment:
There is a lot of code deleted from the master branch which hints that your branch hasn't been rebased to the latest master

Reply:
this is rebased to the latest commit so I'm not sure where all these differences have come from... can add these back manually but it's odd

Reply:
I've manually added the lost stuff back - perhaps I messed the rebase up and rejected the parallel pipeline changes? I think it should be working OK now

Reply:
Thanks, this now all looks good! 

Maybe the reason was that you made your own copy of this repo and it hasn't kept track of the most recent changes?